### PR TITLE
Implement GitHub Actions (SW-537)

### PR DESCRIPTION
**Pull request by TheKeymaster**

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      - master
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.3']
```
Comment (dblaichinger) [inline .github/workflows/lint.yml:15]: Shouldn't the linter run for the lowest php version we support?

**Review by dblaichinger — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (TheKeymaster) [reply .github/workflows/lint.yml:15]: Why should it? The linter is a packagist package, which is installed via composer. The PHP version shouldn't matter.

Comment (dblaichinger) [reply .github/workflows/lint.yml:15]: When the version doesn't matter it's fine.

**Review by dblaichinger — APPROVED** [review]

**Comment by TheKeymaster** [comment]
Closed in favor of #129.

**Pull request closed without merging**